dev/BBimie/Wine quality.ipynb

```diff
@@ -0,0 +1,1828 @@
+       "      <td>0.9938</td>\n",
+       "      <td>3.22</td>\n",
+       "      <td>0.45</td>\n",
+       "      <td>11.0</td>\n",
+       "      <td>6</td>\n",
+       "      <td>False</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   fixed acidity  volatile acidity  citric acid  residual sugar  chlorides  \\\n",
+       "0            7.0              0.27         0.36            20.7      0.045   \n",
+       "1            6.3              0.30         0.34             1.6      0.049   \n",
+       "2            8.1              0.28         0.40             6.9      0.050   \n",
+       "3            7.2              0.23         0.32             8.5      0.058   \n",
+       "4            7.2              0.23         0.32             8.5      0.058   \n",
+       "5            8.1              0.28         0.40             6.9      0.050   \n",
+       "6            6.2              0.32         0.16             7.0      0.045   \n",
+       "7            7.0              0.27         0.36            20.7      0.045   \n",
+       "8            6.3              0.30         0.34             1.6      0.049   \n",
+       "9            8.1              0.22         0.43             1.5      0.044   \n",
+       "\n",
+       "   free sulfur dioxide  total sulfur dioxide  density    pH  sulphates  \\\n",
+       "0                 45.0                 170.0   1.0010  3.00       0.45   \n",
+       "1                 14.0                 132.0   0.9940  3.30       0.49   \n",
+       "2                 30.0                  97.0   0.9951  3.26       0.44   \n",
+       "3                 47.0                 186.0   0.9956  3.19       0.40   \n",
+       "4                 47.0                 186.0   0.9956  3.19       0.40   \n",
+       "5                 30.0                  97.0   0.9951  3.26       0.44   \n",
+       "6                 30.0                 136.0   0.9949  3.18       0.47   \n",
+       "7                 45.0                 170.0   1.0010  3.00       0.45   \n",
+       "8                 14.0                 132.0   0.9940  3.30       0.49   \n",
+       "9                 28.0                 129.0   0.9938  3.22       0.45   \n",
+       "\n",
+       "   alcohol  quality  recommend  \n",
+       "0      8.8        6      False  \n",
+       "1      9.5        6      False  \n",
+       "2     10.1        6      False  \n",
+       "3      9.9        6      False  \n",
+       "4      9.9        6      False  \n",
+       "5     10.1        6      False  \n",
+       "6      9.6        6      False  \n",
+       "7      8.8        6      False  \n",
+       "8      9.5        6      False  \n",
+       "9     11.0        6      False  "
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "wine = pd.read_csv('winequality.csv')\n",
```

Comment:
Plesae load the dataset from the correct relative path in the repo so that it runs without modification